apps/core/event_loop/runner.py

```
@@ -32,35 +36,53 @@ def handle_command(command: Command, queue: list[Message]):
                 uuid_list = [f"{m!s}" for m in new_messages]
-                print(f"New messages: {uuid_list!s}")
+                logger.debug("New messages: %s", uuid_list)
                 queue.extend(new_messages)
         except Exception as e:
-            print(f"Exception handling event {event.__class__.__name__}: {e!s}")
+            logger.exception(
+                "Exception handling event %s (%s) with handler '%s': %s",
+                event.__class__.__name__,
+                event.uuid,
+                handler.__name__,
+                e,
```

Ruff (0.14.6)

Redundant exception object included in `logging.exception` call

(TRY401)